cross-lib/run_enki_on_all.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+while [ -n "$1" ]; do
+  case "$1" in
+    --command) 
+      shift; arg_command=$1
+    ;;
+    *)
+      echo "Invalid argument $1"
+      exit 1
+    ;;
+  esac
+  shift
+done
+
+[[ $arg_command ]] || ( echo "ERROR: A command was not provided. Typical examples are 'all-git-pdf' or 'all-git-web' or 'all-git-epub'" && exit 1 )
+
+all_books="cross-lib/book_data/AUTO_books.txt"
+test -f $all_books || ( echo "ERROR: Book list not found at ${all_books}" && exit 1 )
+
+mkdir -p cross-lib/logs/
+
+# https://stackoverflow.com/a/20983251
+echo_green() { echo -e "$(tput setaf 2)$*$(tput sgr0)"; }
+echo_red() { echo -e "$(tput setaf 1)$*$(tput sgr0)"; }
+
+run_and_log_enki () {
+  echo "  running command $1 on $2 $3"
+  start_time=$(date +%s)
+  ./enki --data-dir ./data/$3-$1 --command $1 --repo openstax/$2 --book-slug $3 --style default --ref main &> cross-lib/logs/$2-$3.txt
+  exit=$?
+  stop_time=$(date +%s)
+  elapsed_formatted="$(date -u -r $(($stop_time - $start_time)) +%T)"
```

Comment:
The date command here did not work for me at all. The -r option pulls the last modified date from a file. This one did work. This creates a date that is `elapsed_time` after epoch and then pulls out the hours, minutes, and seconds.
```bash
elapsed_formatted="$(date --date="@$((stop_time - start_time))" +%H:%M:%S)"
```

Reply:
hmm, this suggestion isn't working for me 😅 i believe it's a matter of [linux](https://man7.org/linux/man-pages/man1/date.1.html) vs [mac](https://ss64.com/osx/date.html) `date`.

have you encountered this sort of issue before? any thoughts on what to do?

Reply:
update: tried [this](https://github.com/openstax/enki/pull/200/files#diff-1aba1be9b3f156d66a15a2ccf9725ff074ff6a413320298f3762a3d811884ddaR33), does that work for you?

Reply:
Sorry for the late reply, I think you need to @ sign to indicate that the time is in second.
```bash
$ date --date="1" +%H:%M:%S
01:00:00
$ date --date="@1" +%H:%M:%S
00:00:01
```